Contributing.md

```diff
@@ -0,0 +1,160 @@
+# Contributing to c2pa-font-ux-js
+
+- [Issues](#issues)
+- [Pull Requests](#pull-requests)
+  - [PR Naming](#pr-naming)
+  - [PR Labels](#pr-labels)
+  - [PR Descriptions](#pr-descriptions)
+  - [Merging](#merging)
+- [Branching](#branching)
+  - [Development branches](#development-branches)
+  - [dev](#dev)
+  - [main](#main)
+- [Style](#style)
+  - [EditorConfig](#editorconfig)
+- [Documentation](#documentation)
+  - [JavaScript](#javascript)
+- [Unit Test Coverage](#unit-test-coverage)
+
+This repository contains Web Components for JavaScript which support the L1, L2,
+and L3 experiences for C2PA.
+
+Following are general guidelines for contributing source to this repo.
+
+All communications regarding this repository are governed by the
+[Monotype Contributor Covenant Code of Conduct](https://github.com/Monotype/.github/blob/main/CODE_OF_CONDUCT.md)
+(MCCCoC).
```

Comment:
Delete the awkward acronym _MCCCoC_. It is not used anywhere else and just adds noise.